projects/schematics/src/ng-add/files/auth-http-config/auth-http-config.module.__ts__

```diff
@@ -27,8 +27,7 @@ export const httpLoaderFactory = (httpClient: HttpClient) => {
           // autoUserInfo: false,
         };
       })
-    )
-    .toPromise();
```

Comment:
Removed because this is deprecated, and `StsConfigHttpLoader` now only accepts an Observable.